pkg/controller/utils/utils.go

```diff
@@ -128,3 +107,36 @@ func Remove(list []string, s string) []string {
 	}
 	return list
 }
+
+// GenerateShortUUID Generates a short UID
+func GenerateShortUUID() string {
```

Comment:
Semantics: we called a 6 char string a  UID in the AWS Federated Account Access controller and that's what the label uses. Thoughts on changing all UUID to UID to match and then the AFAA controller can use this as well? 

https://github.com/openshift/aws-account-operator/blob/master/pkg/controller/awsfederatedaccountaccess/awsfederatedaccountaccess_controller.go#L163-L164

Reply:
Not sure why I didn't do that as I was looking at the AFAA, will fix.